### ISSUE TITLE
Expected loss curves for adversarial losses

### ISSUE THREAD

**Issue by tejaskhot** [researchmm/TTSR#12]
Hello! Thanks for the great work and sharing code. Can you elaborate a bit on how the loss curves are expected to look for the generator and discriminator through training? This would be useful to know if the GAN is training well when training on different datasets.

**Comment by FuzhiYang** [comment]
I'm sorry that I did not pay special attention on this loss curve. But you can use the provided training script to study loss curve where the hyper-parameters are what we used.
